tensorflow/python/ops/math_ops.py

```diff
@@ -4038,8 +4040,7 @@ def add(x, y, name=None):
 
   Args:
     x: A `tf.Tensor`. Must be one of the following types: bfloat16, half,
-      float32, float64, uint8, int8, int16, int32, int64, complex64, complex128,
-      string.
+      float32, float64, uint8, uint16, int8, int16, int32, int64, complex64, complex128.
```

Comment:
similarly, add backticks

Reply:
> similarly, add backticks
code: Tensorflow Version（2.11）

    import tensorflow as tf
    x = tf.constant([1, 2], dtype=tf.int8)
    tf.add(True, x)

result: InvalidArgumentError: Value for attr 'T' of bool is not in the list of allowed values: bfloat16, half, float, double, uint8, uint16, uint32, uint64, int8, int16, int32, int64, complex64, complex128; NodeDef: {{node AddV2}}; Op<name=AddV2; signature=x:T, y:T -> z:T; attr=T:type,allowed=[DT_BFLOAT16, DT_HALF, DT_FLOAT, DT_DOUBLE, DT_UINT8, DT_UINT16, DT_UINT32, DT_UINT64, DT_INT8, DT_INT16, DT_INT32, DT_INT64, DT_COMPLEX64, DT_COMPLEX128]; is_commutative=true; is_aggregate=true> [Op:AddV2]

As can be seen from the above code, the type of parameter x not supports string types.
I will submit later, and I will also add backticks